src/SDKs/Search/DataPlane/Search.Tests/SessionRecords/Microsoft.Azure.Search.Tests.GetAutocompleteTests/TestAutocompleteGetOneTerm.json

```diff
@@ -0,0 +1,68 @@
+{
+  "Entries": [
+    {
+      "RequestUri": "/subscriptions/3c729b2a-4f86-4bb2-abe8-4b8647af156c/resourceGroups/azsmnet4511/providers/Microsoft.Search/searchServices/azs-7938?api-version=2015-08-19",
+      "EncodedRequestUri": "L3N1YnNjcmlwdGlvbnMvM2M3MjliMmEtNGY4Ni00YmIyLWFiZTgtNGI4NjQ3YWYxNTZjL3Jlc291cmNlR3JvdXBzL2F6c21uZXQ0NTExL3Byb3ZpZGVycy9NaWNyb3NvZnQuU2VhcmNoL3NlYXJjaFNlcnZpY2VzL2F6cy03OTM4P2FwaS12ZXJzaW9uPTIwMTUtMDgtMTk=",
```

Comment:
This looks very wrong. There's only a call to ARM to delete the test search service.

Reply:
Thanks for noticing it. It looks like this session was recorded while refactoring was in progress. This test doesn't exist anymore.